packages/app/src/systems/Ecosystem/components/ProjectItem/ProjectItem.tsx

```diff
@@ -0,0 +1,146 @@
+import { cssObj } from '@fuel-ui/css';
+import {
+  Box,
+  ButtonLink,
+  Card,
+  IconButton,
+  Tag,
+  Text,
+  Icon,
+} from '@fuel-ui/react';
+import { motion } from 'framer-motion';
+import type { FC } from 'react';
+
+import type { Project } from '../../types';
+import { STATUS_TEXT } from '../../types';
+
+import { ProjectItemLoader } from './ProjectItemLoader';
+
+import { animations } from '~/systems/Core';
+
+const MotionCard = motion(Card);
+
+export type ProjectItemProps = Project & {
+  onPress?: () => void;
+};
+
+type ProjectItemComponent = FC<ProjectItemProps> & {
+  Loader: typeof ProjectItemLoader;
+};
+
+export const ProjectItem: ProjectItemComponent = ({
+  name,
+  description,
+  url,
+  status,
+}: ProjectItemProps) => {
+  let hostname = '';
+  try {
+    hostname = new URL(url || 'https://#').hostname;
+  } catch (e) {
+    hostname = '';
+  }
+
+  const onCardPress = () => {
+    window.open(url, '_blank');
+  };
+
+  return (
+    <MotionCard
+      withDividers
+      css={styles.card}
+      {...animations.slideInBottom({
+        transition: { type: 'spring' },
+      })}
+      onClick={onCardPress}
+    >
+      <Card.Body>
+        <Box.Flex align="flex-start" justify="flex-start" gap="$4">
+          <IconButton
+            intent="error"
+            variant="ghost"
+            icon="Bolt"
+            aria-label="project-icon"
+            iconSize={20}
+            css={styles.projectIcon}
+          />
+          <Box.Stack gap="$2" css={styles.details}>
+            <Box.Flex align="flex-start" justify="space-between">
+              <Text fontSize="base" color="intentsBase12">
+                {name}
+              </Text>
+              <Icon icon="ArrowUpRight" size={20} stroke={1} />
+            </Box.Flex>
+            <Text fontSize="sm"> {description}</Text>
+            <Box.Flex align="center" justify="space-between" wrap="wrap">
+              <ButtonLink
+                variant="link"
+                css={styles.link}
+                href={url}
+                color="intentsBase12"
+                size="sm"
+              >
+                {hostname}
+              </ButtonLink>
```

Comment:
in this button link previously we had a icon in the button. would be nice to add it again

<img width="167" alt="image" src="https://github.com/FuelLabs/fuels-portal/assets/8636507/76e4d64b-7f5f-4d76-96eb-df1fb962355e">

Reply:
No.
1) it's not in the design. 
2) We already have the arrow icon on the card to indicate that it's a link.
3) The entire card is clickable now, not just the link.